crates/sla/src/lib.rs

```diff
@@ -139,7 +141,11 @@ impl<T: Config> Module<T> {
         let bounded_new_sla = Self::_limit(T::SignedFixedPoint::zero(), new_sla, max_sla);
 
         <VaultSla<T>>::insert(vault_id.clone(), bounded_new_sla);
-        Self::deposit_event(<Event<T>>::UpdateVaultSLA(vault_id, bounded_new_sla));
+        Self::deposit_event(<Event<T>>::UpdateVaultSLA(
+            vault_id,
+            bounded_new_sla,
+            delta_sla,
```

Comment:
Note that `delta_sla` is the _potential_ sla diff - if the vault is already at 100, `delta_sla` can still be > 0